Tests/Microsoft.AppCenter.Crashes.Test.Windows/Utils/ErrorLogHelperTest.cs

```diff
@@ -47,6 +47,9 @@ public void CreateErrorLog(string sessionId)
             // Set correlation identifier like Analytics does to set sessionId.
             SessionContext.SessionId = sessionId == null ? default(Guid?) : Guid.Parse(sessionId);
 
+            var mockUser = "dummyUser";
```

Comment:
Let's use 2 more datarows and a new parameter userId

Then test

null, null
theExampleSid, null
null, theExampleUserId
both

Reply:
Done